di_website/templates/404-static.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<!--[if IE 9]> <html lang="en-GB" class="no-js ie9"> <![endif]-->
+<!--[if (gt IE 9)|!(IE)]><!--><html lang="en-GB" class="no-js"><!--<![endif]-->
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Development Initiatives Pattern Library</title>
+</head>
+<body class="body body--error" id="body-error">
+	<div class="ui-base">
+
+		<!-- ID and -nofocus required for proper skiplink operation -->
+		<main id="pagecontent" class="pagecontent -nofocus" role="main" tabindex="-1">
+            <section class="hero hero--minor">
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <span class="hero__flourish"></span>
+                <div class="row row--narrow">
+                    <div class="hero__content">
+                        <h1 class="hero__title"><span>404 error</span></h1>
+                        <p class="hero__excerpt"><span>What you were looking for cannot be found</span></p>
+                    </div>
+                </div>
+            </section>
+
+            <div class="section">
+                <div class="row row--narrow">
+                    <div class="is-typeset as-wysiwyg max-meter">
+
+                        <h4>Some things that might be helpful</h4>
+                        <ul>
+                            <li>Report this link as broken by emailing <a href="mailto:email@devinigt.org.uk">email@devinit.org.uk</a></li>
+                            <li><a href="/" rel="home">Our homepage</a></li>
+                            <li><a href="#">Contact us</a></li>
```

Comment:
Probably remove this link from the static.

Reply:
Done